themes/ssp/discopower/disco.tpl.php

```diff
@@ -25,20 +25,26 @@
 
 $this->data['header'] = $this->t('selectidp');
 $this->data['header'] = $this->t($this->data['header']);
-$this->data['jquery'] = ['core' => true, 'ui' => true, 'css' => true];
+$this->data['jquery'] = ['core' => true, 'ui' => false, 'css' => false];
 
-
-$this->data['head'] = '<script type="text/javascript" src="'.
-    SimpleSAML\Module::getModuleURL('discopower/assets/js/jquery.livesearch.js').'"></script>'."\n";
-$this->data['head'] .= '<script type="text/javascript" src="'.
-    SimpleSAML\Module::getModuleURL('discopower/assets/js/' . $this->data['score'].'.js')  . '"></script>'."\n";
-
-$this->data['head'] .= $this->data['search'];
+$this->data['head'] = '<link rel="stylesheet" media="screen" type="text/css" href="'.
+    SimpleSAML\Module::getModuleURL('discopower/assets/css/uitheme1.12.1/jquery-ui.min.css').'" />';
```

Comment:
@nikosev why did you have to load ui css theme?

Reply:
@olgabrani I added the CSS to be compatible with the latest version of discopower which supports SimpleSAMLphp v1.18
https://github.com/simplesamlphp/simplesamlphp-module-discopower/blob/release-0.9/templates/disco.tpl.php#L3-L25
Also this change is fixing the loading issue of the languages